contracts/ClaimsManager.sol

```diff
@@ -12,7 +13,7 @@ contract ClaimsManager is
     struct Claim {
         address claimant;
         address beneficiary;
-        uint256 amount;
+        uint256 amountInUsd;
```

Comment:
`amountInUsd` is not expressed in wei and probably never going to be a BigNumber so there might be an optimization opportunity if we move it down a line and pack it along side with `updatedTime` into a single 32 bytes slot, right? Similar thing might be applicable in `Quota`

Reply:
You're right, I did forget to pack this struct. I'll look into it.

Reply:
I used the enum to pack (because it's 8 bits) instead of reducing `amountInUsd` size. I also packed the other two structs.